mods/ts/weapons/otherweapons.yaml

```diff
@@ -66,3 +66,14 @@ Tiberium:
 		Spread: 42
 		Damage: 2
 		DamageTypes: EnergyDeath # TODO: FIX ME!
+
+Veins:
+	ReloadDelay: 16
+	Warhead@Damage: SpreadDamage
+		Spread: 42
+		Damage: 5
+		DamageTypes: EnergyDeath # TODO: FIX ME!
+	Warhead@Effect: CreateEffect
+		Explosions: veins
+		ExplosionPalette: player
+		ValidImpactTypes: Ground
```

Comment:
This makes it display ONLY if the target is 'missed'. You need to add `GroundHit` as well.